public_dropin_gpu_environments/nemo_inference_ms/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# This is the default base image for use with user models and workflows.
+FROM nvcr.io/ohlfw0olaadg/ea-participants/nemollm-inference-ms:24.01
```

Comment:
**TODO** 
This image should be uploaded into our [docker hub
](https://hub.docker.com/orgs/datarobotdev/repositories)